### PR TITLE
Build Works and About Me Sections

### PR DESCRIPTION
**Pull request by amentetteh**
**Added these features:**

- Works section
- About Me section

**Below file are concerned :**
- styles.css is updated
- index.html is also updated
- aboutme-illsutration-mobile.png is added

### CODE REVIEW

**Review by jerryowusu — COMMENTED** [review]
Hi @amentetteh 

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

invalid

## Invalid Code Review Request

You have submitted a project, that has **linter checks not working appropriately** and also **contain merge conflicts**. Kindly fix these issues before submitting for review.

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

File: index.html [hunk 1]
```diff
@@ -59,6 +59,170 @@ <h2 class="primary-header green">
...
+  
+            <ul class="project-tags-list">
+              <li><button class="tag-button">Ruby on rails</button></li>
+              <li><button class="tag-button">css</button></li>
+              <li><button class="tag-button">Javascript</button></li>
+              <li><button class="tag-button">html</button></li>
+            </ul>
+            </p>
+  
+            <button class="green-button" id="detail-button">See Project</button>
```
Comment (ichala) [inline index.html:136]: - [ ] kindly add the button interactions styling: Active, Hover, and Pressed like mentioned in Figma design , please add it to all of your buttons

image

File: index.html [hunk 2]
```diff
@@ -59,6 +59,170 @@ <h2 class="primary-header green">
...
+
+        <p id="about-me-languages">Skills</p>
+        <ul class="about-me-tags-list">
+          <li><button class="tag-button">Codekit</button></li>
+          <li><button class="tag-button">GitHub</button> </li>
+          <li><button class="tag-button">Codepen</button> </li>
+          <li><button class="tag-button">GitLab</button> </li>
+          <li><button class="tag-button">Terminal</button> </li>
+        </ul>
+      </article>
```
Comment (ichala) [inline index.html]: - [ ] to be semantically correct The technology categories ("Languages", "Frameworks" and "Skills") should be in a `ul`, each of them in a `li`, kindly fix that

File: index.html [hunk 3]
```diff
@@ -59,6 +59,170 @@ <h2 class="primary-header green">
...
+        <header><img class="about-me-items-image" src="assets/images/icon-languages.svg" alt="Language"></header>
+
+        <p id="about-me-languages">Languages</p>
+        <ul class="about-me-tags-list">
+          <li><button class="tag-button">Javascript</button></li>
+          <li><button class="tag-button">Ruby</button> </li>
+          <li><button class="tag-button">Html</button> </li>
+          <li><button class="tag-button">CSS</button> </li>
+        </ul>
+      </article>
```
Comment (ichala) [inline index.html]: - [ ] kindly reduce the margins (spaces) between the technologies title and the icon on top of it to match the Figma design

Figma           | Your Design
:-------------------------:|:-------------------------:
image | image

File: index.html [hunk 4]
```diff
@@ -59,6 +59,170 @@ <h2 class="primary-header green">
...
+        <p id="about-me-paragraph">
+          I can help you build a product , feature or website Look through some
+          of my work and experience! If you like what you see and have a project
+          you need coded,
+          don’t hestiate to contact me.
+        </p>
+        <button class="green-button" id="resume-button">Get My Resume</button>
+      </div>
+      <article class="about-me-item">
+        <header><img class="about-me-items-image" src="assets/images/icon-languages.svg" alt="Language"></header>
```
Comment (ichala) [inline index.html]: - [ ] I don't see any use in wrapping your image (logo) inside a `<header>` tag, to make your code cleaner and more professional I would suggest removing the header tag from all the logos of the technologies

**Review by ichala — CHANGES_REQUESTED** [review]
Hi @amentetteh, 

Good job so far! 👏👏👏 
There are some issues that you still need to work on to go to the next project but you're almost there!

the-simpsons-homer-simpson

## To Highlight 🎉

- Your Design is Awesome ✔️ 
- Clean Code✔️

 ## Required Changes ♻️

- [ ] Check the comments below

## Optional suggestions

- N/A

_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html (hunk 1 above)
Comment (amentetteh) [reply index.html:136]: Done

**Comment by amentetteh** [comment]
@ichala, changes done.

File: index.html [hunk 5]
```diff
@@ -59,6 +59,174 @@ <h2 class="primary-header green">
       </p>
       <p id="collapse-action-button"></p>
     </section>
+    <section class="works" id="recent-works">
+      <h1 class="primary-header">
+        My Recent Works
+        <p id="indicator"></p>
+      </h1>
```
Comment (Kamba56) [inline index.html]: - [Optional] Please use only one `<h1>` tag in each html file. Kindly make this another heading tag for example `<h2>`. This is best SEO practice. Kindly check [here](https://yoast.com/one-h1-heading-per-post/) for more information.

**Review by Kamba56 — APPROVED** [review]
Hi @amentetteh,

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 
👏👏👏👏👏👏👏👏👏👏👏👏👏👏👏👏👏👏👏👏👏👏👏👏
## Highlights
- [x] All linters passed

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**